packages/@aws-cdk/aws-codepipeline/lib/stage.ts

```diff
@@ -13,6 +46,14 @@ export interface StageProps {
      * The Pipeline to add the newly created Stage to.
      */
     pipeline: Pipeline;
+
+    /**
+     * Allows specifying where should the newly created {@link Stage}
+     * be placed in the Pipeline.
+     *
+     * @default the stage is added at the end of the Pipeline
+     */
+    placed?: StagePlacement;
```

Comment:
I would call this placement. Our properties are usually nouns and not verbs

Reply:
I named it so that the code reads a little better (`new Stage(this, 'Stage', { placed: { atIndex: 3 }})`), but I can change it to `placement` if you prefer that.

Reply:
As I said, properties are usually nouns and not verbs. Just trying to keep our semantic style uniform

Reply:
Will fix.